src/app/settings.py

```diff
@@ -79,6 +79,14 @@
 
 WSGI_APPLICATION = "app.wsgi.application"
 
+# Django debug toolbar settings
+
+if ENVIRONMENT == "development":
+    INSTALLED_APPS += ["debug_toolbar"]
+    MIDDLEWARE += ["debug_toolbar.middleware.DebugToolbarMiddleware"]
+    INTERNAL_IPS = env.list("INTERNAL_IPS", [])
```

Comment:
If you have [SHOW_TOOLBAR_CALLBACK](https://django-debug-toolbar.readthedocs.io/en/latest/configuration.html#toolbar-options) set as `lambda: True`, you don't need this config

Reply:
```python
def show_toolbar(request):
    if request.is_ajax():
        return False
    return True

DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": "app.settings.show_toolbar"}
```

Reply:
I found this snippet 👆 on the web, not sure what it does though.

Reply:
```
def show_toolbar(request):
    return not request.is_ajax()
```

Reply:
One-liner:
```python
DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda request: not request.is_ajax()}
```